### DISCUSSION TITLE
Is it possible to assist in writing and editing docs?

### DISCUSSION THREAD

**Discussion by OSuwaidi** [JetBrains/lets-plot#980] [Q&A]
If any assistance is required in writing new pages, sections, or rewriting current ones in the [docs](https://lets-plot.org/index.html), I would be more than happy to help!

**Comment by alshan** [comment]
Hi @OSuwaidi ! Thank you for willingness to invest time in improving of our documentation.

If you want to improve / rewrite something in existing pages, than you can fork this Github repository: https://github.com/JetBrains/lets-plot-docs
Then push your updates to it and create a pull-request in the orig. repository (above).

Any idea about new pages is a topic for discussion.

In our immediate plans we wanted to create a rich set of cook-book style notebooks and organize then into a user-guide kind of set of pages - from basics up to advanced topics.

For example, annotating stacked charts (see our recent discussion) is a rather an advanced topic. While your memories are fresh, you could try to compose a cookbook covering this area. 

This directory contains various datasets we are using in demos: https://github.com/JetBrains/lets-plot-docs/tree/master/data

**Reply by OSuwaidi** [reply]
Hey @alshan ,

Sorry I was away on vacation and now I'm back.

Looking forward to working on this cook-book style notebooks you mentioned!

Any specific plans on what needs to be implemented or written?

**Reply by alshan** [reply]
Hi @OSuwaidi 
As you were recently working on stacked bars/texts and as the whole ggplot "stacking" situation is still fresh in your memory, you maybe  can create a cookbook covering this topic? 

But as we haven't yet even started working on cook-books , you really can choose whichever topic you like better. Even designing a master table-of-content for the entire "user guide" (i.e. for a collection of cook-books).

For demo purposes we host a collection of datasets here: https://github.com/JetBrains/lets-plot-docs/tree/master/data 
And we can expand it if necessary.

**Reply by OSuwaidi** [reply]
Hey @alshan ,
thanks for the guide. And yeah sure, that sounds great! But before I delve into the cookbook's details and implementation, covering each distinct topic:
1. is there a particular format that I need to follow (a particular structure, dos and don'ts, etc.)?
2. am I free to use any dataset from the collection you've provided (whatever suits the topic at hand), or do we prefer ones over others?

thanks!

**Comment by alshan** [comment]
>is there a particular format that I need to follow (a particular structure, dos and don'ts, etc.)?

Yes indeed, we do have a cook-book template in the repo: https://nbviewer.org/github/JetBrains/lets-plot/blob/master/docs/cookbook_template.ipynb

IMO, the ToC there is unnecessary.

>am I free to use any dataset 

Yes, absolutely.

BTW, you may want to consider using Datalore as an alternative - it's very easy to share notebooks there and it allows to collaborate when working on a notebook.

**Reply by OSuwaidi** [reply]
Hey @alshan !

Thanks for the suggestion; I've already been using Datalore for a quite some time now, and it's great.

How can I share the notebook(s) on the cookbook's topics with you for your review, suggestions, and feedback?

(P.S. I **really** love the website's new UI and facelift, congrats! 🎊)

**Reply by alshan** [reply]
> I really love the website's new UI and facelift

Great, thanx :)

In Datalore you can add a new workspace and then share it with me using igor.alshan@gmail.com  to send an invite. Than I believe I will be able to see/edit any notebook in this workspace.
